Snowflake.Data/Core/FileTransfer/SFFileTransferAgent.cs

```diff
@@ -527,13 +527,19 @@ internal async Task updatePresignedUrlAsync(CancellationToken cancellationToken)
         /// </summary>
         /// <param name="query">The query containing the file path</param>
         /// <returns>The file path contained by the query</returns>
-        private string getFilePathFromPutCommand(string query)
+        internal static string getFilePathFromPutCommand(string query)
```

Comment:
this method is executed only for GCS. Does it mean that the problem was only with GCS?

Reply:
Yes